### PR TITLE
add a new sub page on updating to dynamo 3.x

### PR DESCRIPTION
**Pull request by mjkkirschner**
Screenshot 2024-04-09 at 6 23 34 PM

### CODE REVIEW

File: 11_developer_primer/3_developing_for_dynamo/7-updating-your-packages-and-dynamo-libraries-for-dynamo-3x-Net8.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+You can read more in our blog post [here](https://dynamobim.org/dynamo-on-net-8/):
+***
+
+### Package Compatibility: <a href="#package-compatibility" id="package-compatibility"></a>
+
+#### Using Dynamo 2.x Packages in Dynamo 3.x 
+Because Dynamo 3.x now runs on the .NET8 runtime packages that were built for Dynamo 2.x (*using .NET48*) are not guaranteed to work in Dynamo 3.x. When you attempt to download a package which was published from a Dynamo version less than 3.0 in Dynamo 3.x you will get a warning that the package is from an older version of Dynamo. 
+
+**This does not mean the package will not work** It's simply a warning that there could be compatibility issues, and in general it's a good idea to check if theres a newer version that has been built specifically for Dynamo 3.x.
```
Comment (hwahlstrom) [inline 11_developer_primer/3_developing_for_dynamo/7-updating-your-packages-and-dynamo-libraries-for-dynamo-3x-Net8.md:19]:
```suggestion
**This does not mean the package will not work** It's simply a warning that there could be compatibility issues, and in general it's a good idea to check if there's a newer version that has been built specifically for Dynamo 3.x.
```

File: 11_developer_primer/3_developing_for_dynamo/7-updating-your-packages-and-dynamo-libraries-for-dynamo-3x-Net8.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+You may also notice this type of warning in your dynamo log files at package load time. If everything is working correctly you can ignore it.
```
Comment (hwahlstrom) [inline 11_developer_primer/3_developing_for_dynamo/7-updating-your-packages-and-dynamo-libraries-for-dynamo-3x-Net8.md:21]:
```suggestion
You may also notice this type of warning in your Dynamo log files at package load time. If everything is working correctly, you can ignore it.
```

**Review by hwahlstrom — APPROVED** [review]
Looks good, made some minor punctuation-type fixes.

**Comment by mjkkirschner** [comment]
thanks for all the edits @hwahlstrom!

**Pull request merged**